### ISSUE TITLE
Freezes on xorcrypt cipher

### ISSUE THREAD

**Issue by hkatzdev** [Ciphey/Ciphey#600]
**⚠️IMPORTANT⚠️ if you do not fill this out, we will automatically delete your issue. We will not help anyone that cannot fill out this template.**

- [X] Have you read our [Wiki page "Common Issues & Their Solutions"?](https://github.com/Ciphey/Ciphey/wiki/Common-Issues-&-Their-Solutions)

**Describe the bug**
Ciphey freezes on `ciphey -t "tJseAuuesGnosMauFYtogoHdHTeYCnaeMAve"` when "Trying xorcrypt cipher".
Seems very similar to issue #395.

**Version**
**⚠️IMPORTANT⚠️ We need this information because different environments will induce different bugs in Ciphey**
 - OS/Distro: Arch
 - Python version: Python 3.9.2
 - Ciphey versions:
```Name: ciphey
Version: 5.11.1
Summary: Automated Decryption Tool
Home-page: None
Author: Brandon
Author-email: brandon@skerritt.blog
License: MIT
Location: /usr/lib/python3.9/site-packages
Requires: click-spinner, pyyaml, cipheycore, base58, flake8, base91, click, appdirs, yaspin, pylint, loguru, rich, cipheydists, pybase62
Required-by: 
---
Name: cipheycore
Version: 0.3.2
Summary: 
Home-page: None
Author: Cyclic3
Author-email: cyclic3.git@gmail.com
License: None
Location: /usr/lib/python3.9/site-packages
Requires: 
Required-by: ciphey
---
Name: cipheydists
Version: 0.3.34
Summary: A collection of distributions, character sets and dictionaries for use in ciphey
Home-page: None
Author: Cyclic3
Author-email: cyclic3.git@gmail.com
License: None
Location: /usr/lib/python3.9/site-packages
Requires: 
Required-by: ciphey
```
I also tried the latest git version (commit hash: 1ebec53cf505dd6eac770cdf15fa0c6e596b5001)
 - Did you use Docker?
No
**Verbose Output**
https://gist.github.com/hkatzdev/fb09bd054a251bd45ecb2023fd580e4e

**To Reproduce**
1. What input did you use?: tJseAuuesGnosMauFYtogoHdHTeYCnaeMAve
2. What flags / arguments did you use?: I tried with -t, -vvv, and -crib. No success with any of them.

**Expected behavior**
For the program to either crack the cipher or say it can't crack it.

**Comment by bee-san** [comment]
Hey, this is fixed in 6.0.0 where we remove the xorcrypt cipher 😅  https://github.com/Ciphey/Ciphey/issues/591

**Comment by hkatzdev** [comment]
Oh, sorry! I'll close this issue then.
